library/portable-simd/crates/core_simd/src/simd/ptr/const_ptr.rs

```diff
@@ -39,7 +39,7 @@ pub trait SimdConstPtr: Copy + Sealed {
     /// This method semantically discards *provenance* and
     /// *address-space* information. To properly restore that information, use [`Self::with_addr`].
     ///
-    /// Equivalent to calling [`pointer::addr`] on each element.
+    /// Equivalent to calling [`pointer::addr_without_provenance`] on each element.
     fn addr(self) -> Self::Usize;
```

Comment:
the portable-simd functions should be renamed too, since they're intended to match the pointer functions

Reply:
portabe-simd has its own repo so I didn't want to do that in this PR.

Reply:
well, imo if you're changing anything in portable-simd, may as well do the full change

Reply:
@workingjubilee @calebzulawski please let me know which way you prefer